disable-comments.php

```diff
@@ -393,7 +393,8 @@ public function settings_menu() {
 	}
 
 	public function settings_page() {
-		include dirname( __FILE__ ) . '/includes/settings-page.php';
+		include dirname( __FILE__ ) . '/includes/disable-comments-form.php';
+		include dirname( __FILE__ ) . '/includes/delete-comments-form.php';
```

Comment:
Good call, I've moved delete comments into a tools page. Under Network Admin however since there's no Tools menu it's under Settings when Network Activated.